Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -77,14 +77,28 @@ namespace Unity.MeshSync
 /// </summary>
 internal delegate void DeleteInstanceHandler(string path);
 
+    public enum UpdateKind {
+        Asset,
+        Entity,
+        Instance,
+        InstanceInfo,
+        Property
+    }
+
+    public struct MeshSyncAnalyticsData {
+        internal AssetType asset_Type;
+        internal EntityType entity_type;
```

Comment:
Can we keep consistent naming convention here with `assetType` and `entityType`

Reply:
asset_Type / entity_type is actually conforming with the code style enforced in MeshSync, but the type was inconsistent with entity_type, so that's been fixed

Reply:
I don't understand where this is enforced in MeshSync - I don't see that casing anywhere else. It's all camelCase.

![image](https://user-images.githubusercontent.com/68228653/192552819-d1c9687b-a99f-47f9-976f-3c33cbc85462.png)

Reply:
I agree. it should be camelCase.

Reply:
The current code is a bit messy, but as a rule of thumb, we are trying to apply this convection in this MeshSync package.
https://confluence.unity3d.com/display/AnimeTemplate/Code+Conventions